webui/react/src/pages/ModelDetails.tsx

```diff
@@ -76,11 +76,11 @@ const ModelDetails: React.FC = () => {
 
   const { canModifyModel, canModifyModelVersion, loading: rbacLoading } = usePermissions();
 
-  const {
-    settings,
-    isLoading: isLoadingSettings,
-    updateSettings,
-  } = useSettings<Settings>(settingsConfig(modelId));
+  const config = useMemo(() => {
+    return settingsConfig(modelId);
+  }, [modelId]);
+
+  const { settings, isLoading: isLoadingSettings, updateSettings } = useSettings<Settings>(config);
```

Comment:
the issue is that `settingsConfig(modelId)` returns a raw object which triggers rendering because `Object.is` always returns `false`